src/components/App.jsx

```diff
@@ -56,4 +64,16 @@ App.propTypes = {
   children: PropTypes.object
 };
 
-export default App;
+const queries = {
+  user: {
+    query: gql`
+      query getCurrentUser {
+        currentUser {
+          darkMode
```

Comment:
Add 'id' here before darkmode to clear up this message:
![image](https://github.com/user-attachments/assets/82c338e2-13fe-4ab9-9b80-975f09e58c16)